UnableToInjectException: Unable to start activity

I was facing this issue multiple times:
```
E/AndroidRuntime: FATAL EXCEPTION: main
E/AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mattprecious.prioritysms/com.mattprecious.prioritysms.activity.Al
armActivity}: butterknife.Views$UnableToInjectException: Unable to inject views for com.mattprecious.prioritysms.activity.AlarmActivity@4141c320
E/AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2306)
[...]
E/AndroidRuntime: Caused by: butterknife.Views$UnableToInjectException: Unable to inject views for com.mattprecious.prioritysms.activity.AlarmActivity@4141c320
E/AndroidRuntime:        at butterknife.Views.inject(Views.java:124)
E/AndroidRuntime:        at butterknife.Views.inject(Views.java:52)
E/AndroidRuntime:        at com.mattprecious.prioritysms.activity.AlarmActivity.onCreate(AlarmActivity.java:157)
E/AndroidRuntime:        ... 15 more
E/AndroidRuntime: Caused by: java.lang.reflect.InvocationTargetException
E/AndroidRuntime:        at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime:        at java.lang.reflect.Method.invoke(Method.java:511)
E/AndroidRuntime:        at butterknife.Views.inject(Views.java:119)
E/AndroidRuntime:        ... 17 more
E/AndroidRuntime: Caused by: java.lang.IllegalStateException: Required view with id '2131165260' for field 'mMessageContainerView' was not found. If this field binding is optional add '@Optional'.
E/AndroidRuntime:        at com.mattprecious.prioritysms.activity.AlarmActivity$$ViewInjector.inject(AlarmActivity$$ViewInjector.java:17)
E/AndroidRuntime:        ... 20 more
```

--- a/prioritysms/src/main/res/layout/alarm_land.xml
+++ b/prioritysms/src/main/res/layout/alarm_land.xml
@@ -44,6 +44,7 @@
         android:maxLines="1"
         />
     <ScrollView
+        android:id="@+id/message_container"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:clipToPadding="false"

